signer/src/bitcoin/utxo.rs

```diff
@@ -1082,115 +1077,63 @@ impl<'a> UnsignedTransaction<'a> {
         }
     }
 
-    /// An OP_RETURN output with the bitmap for how the signers voted for
-    /// the package.
-    ///
-    /// None is only returned if there are no requests in the input slice.
+    /// An OP_RETURN output with encoded withdrawal request IDs.
     ///
-    /// The data layout for the OP_RETURN depends on whether there are
-    /// withdrawal requests in the input slice. If there are withdrawal
-    /// requests then the output data is:
+    /// The `OP_RETURN` output has a generally-accepted 80 bytes available for
+    /// data (it is not constrained by the Bitcoin protocol itself but rather by
+    /// the miners' policy). We use this space to encode the withdrawal request
+    /// IDs.
     ///
-    /// ```text
-    ///  0       2    3     5        21            41
-    ///  |-------|----|-----|--------|-------------|
-    ///    magic   op   N_d   bitmap   merkle root
-    /// ```
+    /// Contradictory to version 0, version 1 of the OP_RETURN output will
+    /// always return an output, even if there are no requests.
```

Comment:
nit: if I'm reading this correctly this is not true: if we have no withdrawals we don't add anything after the version byte, and in v0 similarly we were omitting the merkle root if there were no withdrawals.

Also is a bit strange talking about `even if there are no requests`: a sweep tx will always service at least one request.

Reply:
Yeah I read too quickly, updated the comments in [b02d783](https://github.com/stacks-network/sbtc/pull/1512/commits/b02d7837663c4cef165471cfb46cb1c4615b92ef)